app/services/twilio_api_service.rb

```diff
@@ -85,20 +81,7 @@ def send_twilio_message(sender_number, recipient_number, message, callback_url =
       status_callback: callback_url,
       body: message
     )
-  rescue Twilio::REST::TwilioError => exception
-    @error = exception
-    report_error(exception)
-  end
-
-  def report_error(e)
-    Sentry.capture_message(
-      "Error while processing notification",
-      extra: {
-        exception: e.to_s
-      },
-      tags: {
-        type: "notification-service"
-      }
-    )
+  rescue Twilio::REST::TwilioError
```

Comment:
Yup I think we should definitely be letting the Twilio error pass through un-rescued. We might be losing contextual info that's attached to the Twilio error.